README.md

```diff
@@ -282,3 +283,257 @@ Ihr könnt unterschiedlice Zahlen und Farben anstelle von `30` und `"white"` wä
+
+### Wichtige Konzepte
+
+1. **Selektoren**: Damit wählst du aus, welche HTML-Elemente gestaltet werden sollen.
+2. **Eigenschaften und Werte**: Mit Eigenschaften wie `color` oder `font-size` legst du fest, wie die Elemente aussehen sollen.
+3. **Regeln**: Eine Regel besteht aus einem Selektor, einer Eigenschaft und einem Wert.
+
+### Einfache Beispiele der wichtigsten Selektoren
+
+#### 1. **Element-Selektor**
+
+Hier wird ein bestimmtes HTML-Element ausgewählt.
+
+```css
+p {
+  color: blue; /* Ändert die Textfarbe aller <p>-Elemente auf Blau */
+}
+```
+
+#### 2. **Klassen-Selektor**
+
+Für mehrere Elemente mit der gleichen Klasse.
+
+```css
+.card {
+  border: 1px solid black; /* Fügt allen Elementen mit der Klasse "card" einen schwarzen Rahmen hinzu */
+}
+```
+
+#### 3. **ID-Selektor**
+
+Für ein einzelnes Element mit einer eindeutigen ID.
+
+```css
+#header {
+  background-color: lightgray; /* Ändert den Hintergrund eines Elements mit der ID "header" */
+}
+```
+
+#### 4. **Universal-Selektor**
+
+Wählt alle Elemente aus.
+
+```css
+* {
+  margin: 0; /* Setzt alle Ränder auf 0 */
+}
+```
+
+#### 5. **Kombinatoren**
+
+Für komplexere Auswahlmöglichkeiten:
+
+```css
+div p {
+  font-size: 14px; /* Ändert die Schriftgröße von <p>, das in einem <div> liegt */
+}
+```
+
+### So fügst du CSS hinzu
+
+1. **Inline** (direkt im HTML-Element):
+
+   ```html
+   <p style="color: red;">Text in Rot</p>
+   ```
+
+2. **Im `<style>`-Tag im HTML**:
+
+   ```html
+   <style>
+     h1 {
+       font-family: Arial;
+     }
+   </style>
+   ```
+
+3. **In einer externen Datei**:  
+   Eine `.css`-Datei, die verlinkt wird:
+   ```html
+   <link rel="stylesheet" href="styles.css" />
+   ```
+
+Mit diesen Grundlagen kannst du das Design deiner Webseite Schritt für Schritt verbessern! 😊
+
+💡 [More about CSS on mdn](https://developer.mozilla.org/en-US/docs/Web/CSS)
+
+</details>
+
+<details>
+
+  <summary>Was ist JavaScript</summary>
+
+---
+
+**JavaScript** ist eine Programmiersprache, die Webseiten interaktiv und dynamisch macht. Während **HTML** die Struktur vorgibt und **CSS** das Design gestaltet, sorgt **JavaScript** dafür, dass sich die Webseite **"bewegt"** und auf Benutzeraktionen reagiert.
+
+### Wofür wird JavaScript verwendet?
+
+1. **Interaktivität**  
+   JavaScript ermöglicht Funktionen wie das Anzeigen von Pop-ups, das Einblenden oder Ausblenden von Inhalten oder das Prüfen von Formularen.  
+   Beispiel:
+
+   - Ein Button, der bei einem Klick eine Nachricht zeigt.
+
+2. **Animationen**  
+   Dinge wie Bildergalerien oder animierte Menüs werden oft mit JavaScript erstellt.
+
+3. **Datenverarbeitung**  
+   JavaScript kann Daten laden, speichern und bearbeiten, z. B. von einem Server oder aus einem Formular.
+
+### Ein einfaches Beispiel
+
+Ein Button, der eine Nachricht zeigt, wenn man darauf klickt:
+
+```html
+<button onclick="alert('Hallo!')">Klick mich!</button>
+```
```

Comment:
Ich würde nicht diese art von eventListener Zeigen. Da wir im Workshop schon einen kleinen Eventlistener in einer JS Datei erstellen, würde ich vorschlagen, hier ebenfalls querySelector und addEventListener zu zeigen oder das Beispiel wegzulassen.